promptsource/promptsource.py

```diff
@@ -414,7 +415,9 @@ def show_text(t, width=WIDTH):
                         if len(prompt) > 1:
                             st.write("Y")
                             show_text(prompt[1], width=40)
-
+                        st.write("Augmented Prompt (demo)")
+                        aug_subject = template.jinja.split("|||")[0].replace("{{text}}", "").strip()
```

Comment:
the trickiest part will be somehow here:
1/ parsing the jinja template should be easy (as you've seen, we decided to use jinja to escape protected text `{{"this text should not be changed"}}`) 
2/ augment and reinsert the jinja variables. You mentioned you had some strategy to re-insert at the correct place. is that correct?

Reply:
My apologies for the late reply and wasn't being clear about the changes.
The previous revision was assuming that #49 and related discussions are not done yet, but now I see #49 is closed so the implementation is updated accordingly.